rebar.config

```diff
@@ -4,8 +4,8 @@
   deps,
   [
     %% git dependencies
-    % {cfapi, {git, "https://github.com/harness/ff-erlang-client-api", {tag, "0.1.0-beta.1"}}},
```

Comment:
The developer from TRR forked this and made some changes. Until we can properly vet them, fall back to the known, good version.